packages/grafana-data/src/types/dataFrameTypes.ts

```diff
@@ -13,6 +13,8 @@ export enum DataFrameType {
 
   TimeSeriesMulti = 'timeseries-multi',
 
+  TimeRanges = 'timeranges',
```

Comment:
@adela-almasan -- I think we can remove this and TimeRegion plugin from this PR.  we may want/need a different render style, but lets start with this one

Reply:
Yes - PR cleanup is next in line!